detectable/src/main/java/com/synopsys/integration/detectable/detectables/maven/cli/MavenCodeLocationPackager.java

```diff
@@ -421,6 +429,10 @@ public boolean isGav(String componentText) {
         return false;
     }
 
+    private boolean isEclipsePackage(String group) {
+        return (group.equals("p2.eclipse-plugin") || group.equals("p2.eclipse-feature") || group.equals("p2.p2-installable-unit"));
```

Comment:
It would be nice to have these strings defined as constants either in this class or somewhere where they might also be used in the future.

Reply:
Since we have three short/simple strings, I suggest we refer to them directly at the point of comparison. Keeping these strings contained within this method will be enough to keep the code readable and maintainable. Also depending on the outcome of IDETECT-4344 (where we will see if similar updates will be needed to other detectors that also generate maven style external IDs), we may call this method from other detectors in which case we can make this method public but still keep the strings contained in the body.